source/changelog.txt

```diff
@@ -12,6 +12,17 @@ Release Notes
    :depth: 1
    :class: singlecol
 
+v1.5.2
+------
+
+*Released July 27, 2022*
+
+``mongosh`` now uses :github:`Node.js driver 4.8.1
+</mongodb/node-mongodb-native/releases/tag/v4.8.1>`.
+
+`Full release notes available on JIRA 
+<https://jira.mongodb.org/secure/ReleaseNote.jspa?projectId=17381&version=34143>`__.
+
```

Comment:
(suggestion)
For consistency sake, maybe use a bulleted list markup, even though there's only one item?

Reply:
Thanks for this suggestion. It seemed odd to me to have a bulleted list with only one item. I think my preference is to leave this as-is.